openMHA.cfg

```diff
@@ -9,9 +9,9 @@ mha.fragsize = 48
+mha.transducers.calib_in.fir = [[0.38830 -51.63502 -21.61600 2.13690 -5.86821 6.76826 -3.84954 2.51246];[0.38830 -51.63502 -21.61600 2.13690 -5.86821 6.76826 -3.84954 2.51246]]
 mha.transducers.calib_out.peaklevel = [92.8 92.8]
-mha.transducers.calib_out.fir = [[-0.0008493245925390666 0.212079412419309 -0.2154042301549152 -0.3025302897591955 0.199775619155252 -0.1870229266228919 -0.7938470989741924 0.8603675948896129 0.478909142449632 -0.6085003614478869 0.07361724294587288 -0.2227288472200418 0.5368590771510556 0.09111033102709504 -0.480841985504344 0.4390128813205694 0.1246445045880798 0.1269349882268854 -0.04198884835603302 -0.2353992092127431 0.2510197443337887 0.0874549322159773 -0.02924594207986645 0.03500546824173258 0.02306546293053465 0.02935658669200722 -0.04983617327855945 0.01127518725520369 -0.03557732411817455 -0.08252133997595766 0.09626162395832474 -0.03965187059451832 -0.02978787895722923 -0.02032215684247022 -0.07226667524292203 0.1197853714014079 -0.07509868745444499 -0.06716864521583707 0.01498418308780167 -0.0128555683058882 0.005643946502094473 -0.05468486125562624 -0.03084551017547309 0.01713430717627588 -0.04031926622896171 -0.0001434979760821464 -0.02137092800354339 -0.02993680876827081];[ -0.0008493245925390666 0.212079412419309 -0.2154042301549152 -0.3025302897591955 0.199775619155252 -0.1870229266228919 -0.7938470989741924 0.8603675948896129 0.478909142449632 -0.6085003614478869 0.07361724294587288 -0.2227288472200418 0.5368590771510556 0.09111033102709504 -0.480841985504344 0.4390128813205694 0.1246445045880798 0.1269349882268854 -0.04198884835603302 -0.2353992092127431 0.2510197443337887 0.0874549322159773 -0.02924594207986645 0.03500546824173258 0.02306546293053465 0.02935658669200722 -0.04983617327855945 0.01127518725520369 -0.03557732411817455 -0.08252133997595766 0.09626162395832474 -0.03965187059451832 -0.02978787895722923 -0.02032215684247022 -0.07226667524292203 0.1197853714014079 -0.07509868745444499 -0.06716864521583707 0.01498418308780167 -0.0128555683058882 0.005643946502094473 -0.05468486125562624 -0.03084551017547309 0.01713430717627588 -0.04031926622896171 -0.0001434979760821464 -0.02137092800354339 -0.02993680876827081]]
+mha.transducers.calib_out.fir = [[-0.0051100 0.0058418 -0.0029015 -0.0075854 0.0054159 -0.0065281 -0.0138712 0.0280986 -0.0101577 -0.0018500 -0.0015728 0.0011131 0.0135029 -0.0179051 0.0148077 -0.0026030];[-0.0051100 0.0058418 -0.0029015 -0.0075854 0.0054159 -0.0065281 -0.0138712 0.0280986 -0.0101577 -0.0018500 -0.0015728 0.0011131 0.0135029 -0.0179051 0.0148077 -0.0026030]]
```

Comment:
... and these as well.

Reply:
-0.00248576   0.00427128  -0.00354440  -0.00515774   0.00339607
 -0.00750198  -0.00889310   0.02055027  -0.00295105  -0.00612312  -0.00140463   0.00479821   0.00721842   -0.01088499   0.00897386   0.00082907

Reply:
Hier ist N=16 und D=8. Auch berechnet den gepushten Skripten. Ich werde die Diskussion und Bilder überarbeiten sobald wir wissen, ob die Filterkoeffizienten passen.